src/main/java/com/dezorganizacja/bridge/view/graphical/CreditsController.java

```diff
@@ -1,10 +1,66 @@
 package com.dezorganizacja.bridge.view.graphical;
 
+import com.dezorganizacja.bridge.presenter.MainPresenter;
+import javafx.animation.PathTransition;
+import javafx.animation.Timeline;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
 import javafx.event.ActionEvent;
+import javafx.event.EventHandler;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Insets;
+import javafx.scene.Parent;
+import javafx.scene.control.Label;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.*;
+import javafx.scene.text.Font;
+import javafx.util.Duration;
+
+import java.io.IOException;
+
+import static java.lang.Thread.sleep;
 
 public class CreditsController extends Controller {
 
+    public AnchorPane creditsPane;
+
     public void closeCredits(ActionEvent actionEvent) {
         presenter.setState("main");
     }
+
+    @Override
+    public void init(MainPresenter presenter) {
+        super.init(presenter);
+
+        FXMLLoader loader = new FXMLLoader(getClass().getResource("/fxml/movingCredits.fxml"));
+        VBox creditsBox;
+        try {
+            creditsBox = loader.load();
+        } catch (IOException e) {
+            throw new RuntimeException("error while loading /fxml/movingCredits.fxml resource", e);
+        }
+
+
+        creditsPane.setBackground(new Background(new BackgroundFill(Color.BLACK, CornerRadii.EMPTY, Insets.EMPTY)));
+
+
+
+        creditsPane.getChildren().add(creditsBox);
```

Comment:
nie wiem, czy nie jepiej by było wsadzić to jako dziecko creditsPane w fxml-u i wziąć tylko referencję